### PR TITLE
You can kill germs by actually cooking the food now (or by setting it on fire).

### PR DESCRIPTION
**Pull request by Ghommie**
## About The Pull Request
It always seemed weird to me that you can kill germs by washing the food in a sink but not by cooking it to be frank. Also fire should work too, though that's not a worksafe strategy.

## Why It's Good For The Game
It makes perfect sense for heat to kill germs. It can also be a fun alternative to using a sink.

## Changelog

:cl:
qol: You can kill germs by actually cooking the food (i.e. frying and grilling) now, or by setting it on fire.
/:cl:

### CODE REVIEW

**Comment by Wallemations** [comment]
> It always seemed weird to me that you can kill germs by washing the food in a sink

Oh god those white mom cooking videos are part of the game

**Comment by Xackii** [comment]
You don’t need to wash food before cooking because high temperature will clean all the dirt?

File: code/datums/components/food/germ_sensitive.dm [hunk 1]
```diff
@@ -116,9 +119,9 @@ GLOBAL_LIST_INIT(floor_diseases, list(
...
 	parent.AddComponent(/datum/component/infective, new random_disease, weak = TRUE)
 
-/datum/component/germ_sensitive/proc/wash()
+/datum/component/germ_sensitive/proc/delete_germs()
 	SIGNAL_HANDLER
-	if(infective)
+	if(!infective)
 		infective = FALSE
 		qdel(parent.GetComponent(/datum/component/infective))
 		return COMPONENT_CLEANED
```
Comment (00-Steven) [inline code/datums/components/food/germ_sensitive.dm:127]: Is this `infective` check supposed to get inverted? It's a bit odd to set `infective = FALSE` and remove the infective component when `!infective` notes `infective` was already `FALSE`

Comment (00-Steven) [reply code/datums/components/food/germ_sensitive.dm]: cool cool

**Pull request merged**